docs/features/filtering.md

```diff
@@ -209,6 +209,27 @@ Under the hood Fae uses [Kaminari](https://github.com/amatsuda/kaminari) to powe
 * ignore `fae_paginate` and use Kaminari's `paginate` directly
 * etc...
 
+### Enable Pagination in the Controller
+
+Pagination is disabled in the Fad:BaseController by default. This can be customized with a private use_pagination method in controllers inheriting from Fae::BaseController.
```

Comment:
what's this Fad app you speak of? Also, can you put ticks around the `use_pagination` method?